trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc

```diff
@@ -0,0 +1,50 @@
+== Die Leistungsfähigkeit der Organisation erhalten
+
+In der Einführung wurde darauf hingewiesen, dass _Trusted Committers_ sowohl technikorientierte als auch organisationsorientierte Verantwortlichkeiten haben.
+Es ist nicht ausreichend, sich nur auf den _Code_ und den Zustand des _Codes_ zu konzentrieren. Um auf lange Sicht erfolgreich zu sein, sollten _Trusted Committer_ auch danach streben, die 
+Leistungsfähigkeit und Motivation der Organisation, welche die Software erstellt, aufrechtzuerhalten. 
+Deshalb müssen sie sich um eine gute Balance zwischen dem Sicherstellen der Produktqualität und der Weiterentwicklung der Organisation bemühen.
+
+Was macht eine gute Organisation aus? Sehr einfach, in einer guten Organisation neigen die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] dazu dabeizubleiben, 
+sie können den Großteil ihrer Zeit nutzen, um Software zu entwickeln, und sind in der Lage, ihre Fähigkeiten zu verbessern.
+Infolgedessen wird sich eine gute Organisation kontinuierlich selbst verbessern.
+
+Warum schließen sich https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] einer Organisation an und bleiben dabei?
+Einige tun dies, weil sie sich dem Zweck oder der Mission der Organisation anschließen. Es ist die Aufgabe des _Trusted Committers_, diesen Zweck klar zu formulieren und voranzutreiben.
```

Comment:
Change to plural and not singular for the TCs here too?

Reply:
done